### ISSUE TITLE
win_copy does not support large files

### ISSUE THREAD

**Issue by u625030** [ansible/ansible#32677]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

win_copy

##### ANSIBLE VERSION

```
ccbm@amslabweb03.uniface.m4.local> ansible --version
ansible 2.4.1.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/h/CCbm/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python2.7/site-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.5 (default, Aug  4 2017, 00:39:18) [GCC 4.8.5 20150623 (Red Hat 4.8.5-16)]
```

##### CONFIGURATION

DEFAULT_ROLES_PATH(/etc/ansible/ansible.cfg) = [u'/etc/ansible/roles', u'/usr/share/ansible/roles']

##### OS / ENVIRONMENT

N/A (Linux XXXXXXXXXXX.XXXXXXX.XX.XXXXX 3.10.0-327.18.2.el7.x86_64 #1 SMP Thu May 12 11:03:55 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux
)

##### SUMMARY

win_copy by default does not use large file support when building a ZIP file that needs to be transferred. 
When doing a copy of a file larger than 4 Gb, this results in an error. 

##### STEPS TO REPRODUCE

```yaml
  - name: Copy Visual Studio installer to local machine
    win_copy:
      src: /cwnl/common/software/microsoft/VisualStudio_2015
      dest: C:\Temp\BuildMachineTemp\
```

##### EXPECTED RESULTS

File copy should work without problems
##### ACTUAL RESULTS

```
<!--- What actually happened? If possible run with extra verbosity (-vvvv) -->
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/ansible/plugins/action/win_copy.py", line 290, in _copy_zip_file
    zip_file = self._create_zip_tempfile(files, directories)
  File "/usr/lib/python2.7/site-packages/ansible/plugins/action/win_copy.py", line 252, in _create_zip_tempfile
    zip_file.write(file_path, encoded_path, zipfile.ZIP_DEFLATED)
  File "/usr/lib64/python2.7/zipfile.py", line 1137, in write
    self._writecheck(zinfo)
  File "/usr/lib64/python2.7/zipfile.py", line 1102, in _writecheck
    raise LargeZipFile("Filesize would require ZIP64 extensions")
LargeZipFile: Filesize would require ZIP64 extensions
fatal: [amsbldw32i01.uniface.m4.local]: FAILED! => {
    "changed": true,
    "dest": "C:\\Temp\\BuildMachineTemp\\",
    "failed": true,
    "msg": "failed to create tmp zip file: Filesize would require ZIP64 extensions",
    "operation": "folder_copy",
    "src": "/cwnl/common/software/microsoft/VisualStudio_2015"
}
```

**Comment by ansibot** [comment]
cc @dagwieers @jborean93 @jhawkesworth
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by jborean93** [comment]
I swore I tested this but will try again and see what happens. A zip is only used when there are multiple files that need to be changed, if it is just the single file it will be copied across as is.

Edit: I've added some more comments to your PR, I see where I went wrong with my original testing. While it not perfect, while people wait for a release, instead of copying a directory, if you copy a single file a zip file is not used at all and so shouldn't have this issue.
